develop-env/develop_py36/salobj_4/Dockerfile

```diff
@@ -108,7 +108,7 @@ RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib && \
     setup ts_ATMCSSimulator -t current && \
     eups declare -r . -t current && \
     setup ts_standardscripts -t current && \
-    pytest --color=no
+    pytest --color=no || pytest --last-failed
```

Comment:
This seems like a no op if --no-cache is used during the build.

Reply:
What happens here is that, if the first pytest fail, the second one runs... If the first one succeeds the second is ignored.